harpy/common/summaries.py

```
@@ -29,18 +29,18 @@ def align_bwa(self):
-        bwa_static = "-C -v 2" if is_standardized else "-v 2"
+        bwa_static = "-y -x sr" if is_standardized else "-x sr"
         extra   = extra
 
         align = "Sequences were aligned with BWA using:\n"
-        align += f'\tbwa mem {bwa_static} {extra} -R "@RG\\tID:SAMPLE\\tSM:SAMPLE" genome forward_reads reverse_reads |\n'
+        align += f'\tminibwa map {bwa_static} {extra} -R "@RG\\tID:SAMPLE\\tSM:SAMPLE" genome forward_reads reverse_reads |\n'
         align += f"\tsamtools view -h {unmapped} -q {quality}"
         duplicates = "Duplicates in the alignments were marked following:\n"
         duplicates += "\tsamtools collate |\n"
         duplicates += "\tsamtools fixmate |\n"
         duplicates += f"\tsamtools sort -T SAMPLE -m 2000M |\n"
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)